src/plugins/lvm-dbus.c

```diff
@@ -2445,6 +2446,29 @@ gboolean bd_lvm_lvresize (const gchar *vg_name, const gchar *lv_name, guint64 si
     return call_lv_method_sync (vg_name, lv_name, "Resize", params, NULL, extra, TRUE, error);
 }
 
+/**
+ * bd_lvm_lvrepair:
+ * @vg_name: name of the VG containing the to-be-repaired LV
+ * @lv_name: name of the to-be-repaired LV
+ * @pv_list: (array zero-terminated=1): list of PVs to be used for the repair
+ * @extra: (nullable) (array zero-terminated=1): extra options for the LV repair
+ *                                                 (just passed to LVM as is)
+ * @error: (out) (optional): place to store error (if any)
+ *
+ * Returns: whether the @vg_name/@lv_name LV was successfully repaired or not
+ *
+ * Tech category: %BD_LVM_TECH_BASIC-%BD_LVM_TECH_MODE_MODIFY
+ */
+gboolean bd_lvm_lvrepair (const gchar *vg_name, const gchar *lv_name, const gchar **pv_list, const BDExtraArg **extra, GError **error) {
+  (void) vg_name;
```

Comment:
yay, a victim of `-Werror`? :face_with_head_bandage:

Reply:
Oops, I completely missed this. Please use the `UNUSED` macro, we are already using it in lvm-dbus.

Reply:
Oh, much better, thanks!  Done.